Management/DomainModels/AbbrToStateDict.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
```

Comment:
Please consider consolidate this one and another dictionary class to the following static class:

```
using Management.DomainModels;
using System.Collections.Generic;

namespace Management
{
    public static class AbbreviationToState
    {
        private static readonly Dictionary<string, State> _stateLookUp = new Dictionary<string, State>
        {
            { "CA", State.Wrap("California") },
            { "GA", State.Wrap("Georgia") },
            { "IL", State.Wrap("Illinois") },
            { "MA", State.Wrap("Massachusetts") },
            { "WA", State.Wrap("Washington") },
        };

        private static readonly Dictionary<string, List<City>> _cityLookUp = new Dictionary<string, List<City>>()
        {
            { "CA", new List<City>() { City.Wrap("Los Angeles"), City.Wrap("San Diego"), City.Wrap("San Jose"), City.Wrap("San Francisco"), City.Wrap("Fresno") } },
            { "GA", new List<City>() { City.Wrap("Atlanta"), City.Wrap("Savannah"), City.Wrap("Athens"), City.Wrap("Roswell"), City.Wrap("Marietta") } },
            { "IL", new List<City>() { City.Wrap("Chicago"), City.Wrap("Naperville"), City.Wrap("Joliet"), City.Wrap("Rockford"), City.Wrap("Springfield") } },
            { "MA", new List<City>() { City.Wrap("Boston"), City.Wrap("Worcester"), City.Wrap("Springfield"), City.Wrap("Brockton"), City.Wrap("Quincy") } },
            { "WA", new List<City>() { City.Wrap("Seattle"), City.Wrap("Spokane"), City.Wrap("Tacoma"), City.Wrap("Vancouver"), City.Wrap("Bellevue") } },
        };

        public static State GetStateFullName(string abbreviation)
            => _stateLookUp[abbreviation];

        public static IEnumerable<City> GetSupportedCities(string countryCode)
            => _cityLookUp[countryCode];
    }
}
```

Reason: 

1. You are not accessing any instance, the data you are trying to access is "fixed"
2. Inilize instace in multiple class will create a lot lot of overhead, and is not necessary